### PR TITLE
Cleanup cached and unnecessary files

### PR DESCRIPTION
**Pull request by zouyee**
Cleanup cached and unnecessary files

### CODE REVIEW

**Comment by ImJasonH** [comment]
@vbatts @vdemeester (aka "The Vincents"): does this change seem okay to you?

File: buildah/Dockerfile [hunk 1]
```diff
@@ -35,6 +35,14 @@ RUN \
   git checkout "${BUILDAH_REVISION}" && \
   make && \
   make install && \
+  yum clean all && \
+  rm -rf \
+      /var/cache/yum \
+      /usr/share/doc \
+      /usr/share/man \
+     /usr/share/info \
+     /usr/share/locale \
```
Comment (vbatts) [inline buildah/Dockerfile]: hmm i guess locale missing won't affect the outside of the build.

**Comment by vbatts** [comment]
LGTM

I thought we already were `clean`'ing, but that might have been dropped at some point

**Comment by vdemeester** [comment]
@vbatts ahah yeah, never got merge :upside_down_face: :roll_eyes: :stuck_out_tongue_closed_eyes:

**Pull request closed without merging**